### PR TITLE
Enable Hebrew, French, Dutch

### PR DESCRIPTION
**Pull request by jonaharagon**
## ⚠️ **Do not make any changes to files in this PR!!** ⚠️

Fixes for files here **must be made in Crowdin** and changes will be pulled from there.

Please read the warnings in the Netlify deploy log (comment below) to find translation errors.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -95,6 +94,52 @@ plugins:
...
+        'Advanced Topics': 'HEAdvanced Topics'
+        'Recommendations': 'HERecommendations'
+        'Internet Browsing': 'HEInternet Browsing'
+        'Operating Systems': 'HEOperating Systems'
+        'Providers': 'HEProviders'
+        'Software': 'HESoftware'
+        'About': 'HEAbout'
+        'Changelog': 'HEChangelog'
+        'Forum': 'HEForum'
+        'Blog': 'HEBlog'
```
Comment (jonaharagon) [inline mkdocs.yml:112]: These strings need to be translated to Hebrew (manually in this PR)

File: mkdocs.yml [hunk 2]
```diff
@@ -95,6 +94,52 @@ plugins:
...
+        'Advanced Topics': 'FRAdvanced Topics'
+        'Recommendations': 'FRRecommendations'
+        'Internet Browsing': 'FRInternet Browsing'
+        'Operating Systems': 'FROperating Systems'
+        'Providers': 'FRProviders'
+        'Software': 'FRSoftware'
+        'About': 'FRAbout'
+        'Changelog': 'FRChangelog'
+        'Forum': 'FRForum'
+        'Blog': 'FRBlog'
```
Comment (jonaharagon) [inline mkdocs.yml:127]: These strings need to be translated to French (manually in this PR)

File: mkdocs.yml [hunk 3]
```diff
@@ -95,6 +94,52 @@ plugins:
...
+        'Advanced Topics': 'NLAdvanced Topics'
+        'Recommendations': 'NLRecommendations'
+        'Internet Browsing': 'NLInternet Browsing'
+        'Operating Systems': 'NLOperating Systems'
+        'Providers': 'NLProviders'
+        'Software': 'NLSoftware'
+        'About': 'NLAbout'
+        'Changelog': 'NLChangelog'
+        'Forum': 'NLForum'
+        'Blog': 'NLBlog'
```
Comment (jonaharagon) [inline mkdocs.yml:142]: These strings need to be translated to Dutch (manually in this PR)

**Comment by dngray** [comment]
We want to avoid pushing a release with the translations now they've been pushed to main.

Just keep in mind I've pushed this to main https://github.com/privacyguides/privacyguides.org/commit/c6161ef1677d46af7d5389e210d5875ed939c7ad which will be there on release, even if it doesn't show exactly like that on Crowdin.

**Review by dngray — APPROVED** [review]

**Review by freddy-m — APPROVED** [review]
Thanks to all the teams behind these translations! It's so good to see!

**Review by blacklight447 — APPROVED** [review]

**Pull request merged**